### ISSUE TITLE
"cliff.toml" is not found

### ISSUE THREAD

**Issue by sporto** [orhun/git-cliff-action#5]
Hi

One of the commits on the last three days is causing git-cliff to not find the configuration file. I'm seeing this

```
+ git-cliff --config=cliff.toml --verbose
 WARN  git_cliff > "cliff.toml" is not found, using the default configuration.
```

So it is generating an incorrect changelog.
If I manually peg the action to an earlier commit hash, it works well.

Thanks

**Comment by sporto** [comment]
I tried different commit hashes. I believe that this commit https://github.com/orhun/git-cliff-action/commit/5352ef7a0aedf5b706696bc9c1a5e7fcacdf0ceb introduced the issue

**Comment by sebastianpfischer** [comment]
+1 here too

**Comment by orhun** [comment]
I reverted the `v1` tag until I'm available for fixing the issue.

**Comment by orhun** [comment]
Should be fixed in b95829e081e8174b73024a39a60c81e7a070a1fb
